daiquiri/query/settings.py

```diff
@@ -67,6 +67,20 @@
         }
     }
 ]
+QUERY_DOWNLOADS = [
+    {
+        'key': 'table',
+        'model': 'daiquiri.query.models.DownloadJob',
+        'template': 'query/query_download_table.html',
+        'params': ['format_key']
+    },
+    {
+        'key': 'archive',
+        'model': 'daiquiri.query.models.QueryArchiveJob',
+        'template': 'query/query_download_archive.html',
+        'params': ['column_name']
+    }
+]
```

Comment:
What are `QueryArchiveJobs`? Is that meant for `.zip/.tar` download ?

Reply:
Why do we need that? Do you have a specific usage in mind? I though this would be covered by datalink.